vavr/src/main/java/io/vavr/collection/HashArrayMappedTrie.java

```diff
@@ -433,7 +448,7 @@ public int size() {
 
         @Override
         public int hashCode() {
-            return Objects.hash(hash, value);
+            return HashCodes.hash(hash, value);
```

Comment:
this method will be removed, see #2187

Reply:
Thank you for the heads-up. I resolved the merge conflict and updated the PR.